docs/tutorials/start-a-private-network-with-substrate.md

```diff
@@ -9,29 +9,33 @@ You will need:
-
-### Your First Time
-First we'll download the Substrate v1.0 code by cloning the git repository.
 ```bash
-git clone -b v1.0 https://github.com/paritytech/substrate
+# Download the Substrate code
+git clone https://github.com/paritytech/substrate
 cd substrate
+
+# Switch to the proper commit
+git checkout 7d7e74fb
+
+# Install rust prerequisites
+curl https://getsubstrate.io -sSf | bash -s -- --fast
 ```
 
-The repository includes a utility called `subkey` which we'll optionally  use to generate and inspect keypairs. Let's compile it first. The `--force` option means that we'll install this version over any previously installed versions.
+The repository includes a utility called `subkey` which we'll optionally use to generate and inspect keypairs. Let's compile it first. The `--force` option means that we'll install this version over any previously installed versions.
 ```bash
+# Compile subkey
 cargo install --force --path subkey subkey
 ```
 
-Now let's compile the actual blockchain node that we'll be running. Because Substrate is a framework, most real-world blockchains that use it will write custom runtime code. There are [other tutorials](/tutorials/) that cover that process in detail. Luckily the Substrate repository itself already comes with two ready-to-run node environments. The first lives in the `node` directory and includes many features to be a practical blockchain. In fact it looks quite similar to [Polkadot](https://polkadot.network) which is also built on Substrate. The second is a minimalistic runtime that lives in the `node-template` directory. We'll be using the node template in this tutorial because of its simplicity and because it is the usual starting point when writing your own custom runtime.
+Now let's compile the actual blockchain node that we'll be running. Because Substrate is a framework, most real-world blockchains that use it will write custom runtime code. There are [other tutorials](/tutorials/) that cover that process in detail. Luckily the Substrate repository itself already comes with two ready-to-run node environments. The first lives in the `node` directory and includes many features to be a practical blockchain. In fact it looks quite similar to [Polkadot](https://polkadot.network) which is also built on Substrate. The second is a more minimal runtime that lives in the `node-template` directory. We'll be using the node template in this tutorial because of its simplicity and because it is the usual starting point when writing your own custom runtime.
```

Comment:
What, aside from the "simplicity" makes `node-template` the usual starting point of writing a runtime?

Reply:
I think the fact that it was designed to be a good starting point is why we usually start with it. Is there some text you'd like added here?